redisenterprise/README.md

```diff
@@ -1,4 +1,6 @@
-# Redis Enterprise
+# Redis Enterprise (Deprecated)
+
+** This integration is will be deprecated starting on July 19, 2024. Please use the latest [Redis Enterprise Datadog Integration][13] going forward. This new integration exposes all of the latest Redis Enterprise metrics and includes updated dashboards.**
```

Comment:
Please move this to the overview section of the README file

Reply:
Done!